librz/core/cmd_descs/cmd_help.yaml

```diff
@@ -0,0 +1,559 @@
+            arg_str: "??x"
+            comment: Won"t do anything
+  - name: "?-"
+    summary: Execute given command if $? register is less than 0
+    cname: exec_cmd_if_core_num_value_negative
+    args:
+      - name: cmd
+        type: RZ_CMD_ARG_TYPE_CMD
+    details:
+      - name: Examples
+        entries:
+          - text: "?=1; ?-"
+            arg_str: "??x"
+            comment: Won"t do anything
+          - text: "?=-2; ?-"
+            arg_str: "??x"
+            comment: Will display hexdump at current seek address
+          - text: "?=0; ?+"
+            arg_str: "??x"
+            comment: Won"t do anything
+  - name: "?!"
+    summary: Execute command if result of last numeric expression evaluation (related) command was 0
+    cname: exec_cmd_if_core_num_value_zero
+    args:
+      - name: cmd
+        type: RZ_CMD_ARG_TYPE_CMD
+  - name: "?&"
+    summary: Show help menu for task commands
+    cname: show_help_tasks
+    args: []
+  - name: "?%?"
+    summary: Show help for environment variable commands
+    cname: show_help_percent
+    args: []
+  - name: "?$"
+    summary: Help variable commands
+    type: RZ_CMD_DESC_TYPE_GROUP
+    subcommands:
+      - name: "?$"
+        summary: Show help variables
+        cname: show_help_vars
+        args: []
+      - name: "?$?"
+        summary: Show this help message
+        cname: show_help_dollar
+        args: []
+  - name: "?V"
+    summary: Show version information
+    type: RZ_CMD_DESC_TYPE_GROUP
+    subcommands:
+      - name: "?V"
+        summary: Show version info
+        cname: show_version_info
+        args: []
+      - name: "?Vc"
+        summary: Show numeric version
+        cname: show_version_numeric
+        args: []
+      - name: "?Vj"
+        summary: Show version info in JSON format
+        cname: show_version_json
+        args: []
+      - name: "?Vn"
+        summary: Show numeric version
+        cname: show_version_numeric2
+        args: []
+      - name: "?Vq"
+        summary: Quite mode version info
+        cname: show_version_quiet_mode
+        args: []
+      - name: "?V0"
+        summary: Show major version
+        cname: show_version_major
+        args: []
+      - name: "?V1"
+        summary: Show minor version
+        cname: show_version_minor
+        args: []
+      - name: "?V2"
+        summary: Show patch version
+        cname: show_version_patch
+        args: []
+  - name: "?l"
+    summary: String length calculation commands
+    subcommands:
+      - name: "?l"
+        summary: Calculate length of given string and store in $? register
+        cname: compute_string_length
+        args:
+          - name: str
+            type: RZ_CMD_ARG_TYPE_STRING
+      - name: "?lq"
+        summary: Calculate length of given string and store in $? register but print nothing (quietly).
+        cname: compute_string_length_quiet
+        args:
+          - name: str
+            type: RZ_CMD_ARG_TYPE_STRING
```

Comment:
for these kinds of commands you can use something like https://github.com/rizinorg/rizin/blob/dev/librz/core/cmd_descs/cmd_write.yaml#L318 . You just have to specify both the STANDARD and QUIET mode and then implement them in the same handler (like https://github.com/rizinorg/rizin/blob/dev/librz/core/cio.c#L802).